packages/extension/src/libs/networks-state/index.ts

```
@@ -85,16 +134,37 @@ class NetworksState {
     }
   }
 
+  async getEnabeledTestNetworks(): Promise<string[]> {
+    await this.insertNetworksWithNewFeatures();
+    const state: IState | undefined = await this.getState();
+    if (state && state.enabledTestNetworks) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)